packages/appium/docs/en/ecosystem/build-drivers.md

```diff
@@ -566,11 +566,11 @@ create and release client-side plugins for each language you want to support (di
 examples can be found at the relevant client docs).
 
 An alternative to this way of doing things is to overload a command which all WebDriver clients
```

Comment:
I call these "driver extensions" in README

Reply:
since we use the word "extension" to mean something different (a driver or plugin), maybe we could come up with something else. also, drivers can add new commands via the `extraMethodMap` strategy as well.